### ISSUE TITLE
GIVstream Tweet thread

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-planning#579]
From: https://docs.google.com/document/d/1IzvuwLJpsvdNQb3HxcoaRvSTuACr6jAnPHXDPmF8ilM/edit#

**Comment by jfbtheii** [comment]
Draft complete! LMK what you think @laurenluz 
https://docs.google.com/document/d/1kaK_1zSdNvulQ2U6xJVZXXBSDLG5oOY44w4QYt6DL4o/edit

**Comment by laurenluz** [comment]
Thanks @jfbtheii - made a bunch of edits as suggestions. would be awesome if you could go through and make note of them (for next time) when you accept them.

It would be awesome as well if you could help create some memes to share w the article - I left comments wrt to which ones.

Also general note: if you're making threads out of an article, try to shorten the sentences and phrases. a lot of the copy/paste ends up too wordy for twitter. also, I would suggest to first read the article... then consider the most important points to highlight and then make the thread from there. The thread doesn't have to be an exact mirror of the article. It is more impactful if the thread is like a summary - but hits only the most important & most interesting points.

Anyway, as usual, I appreciate your efficiency & ideas!

**Comment by jfbtheii** [comment]
Got it. I appreciate the advice on how to better approach the content of the thread. I'll dive into the edits!

**Comment by jfbtheii** [comment]
Touched up your edits, created some original memes @laurenluz

**Comment by laurenluz** [comment]
Hahah 2 chains is so funny 🤣 Thanks @jfbtheii! This is great!

**Comment by laurenluz** [comment]
FYI you should post any memes in the meme channel on discord :)

**Comment by laurenluz** [comment]
https://twitter.com/Givethio/status/1506295461963083783
